scripts/irods/test/test_iadmin.py

```diff
@@ -2133,8 +2133,14 @@ def test_moduser_type_rodsgroup(self):
         self.assertEqual('rodsuser', lib.get_user_type(self.admin, self.username))
 
     def test_moduser_bad_downgrade(self):
+        SYS_NOT_ALLOWED = -169000
         self.assertEqual('rodsadmin', lib.get_user_type(self.admin, 'rods'))
-        self.admin.assert_icommand(['iadmin', 'moduser', 'rods', 'type', 'rodsuser'])
+        res, _ = lib.execute(['iadmin', 'moduser', 'rods', 'type', 'rodsuser'])
```

Comment:
i think it can just be...

```
self.admin.assert_icommand(['iadmin', 'moduser', 'rods', 'type', 'rodsuser'], 'STDERR_SINGLELINE', 'SYS_NOT_ALLOWED')
```

?